www/src/components/repos/Docker.tsx

```diff
@@ -107,14 +107,27 @@ function ImageVersionPicker({ image }: any) {
   const navigate = useNavigate()
   const { pathname } = useLocation()
   const { dockerRepository } = image
-  const { data, loading } = useQuery(DOCKER_IMG_Q, {
-    variables: { dockerRepositoryId: dockerRepository.id },
+  const [cursor, setCursor] = useState(null)
+  const [allImages, setAllImages] = useState<any[]>([])
+  const { data } = useQuery(DOCKER_IMG_Q, {
+    variables: {
+      dockerRepositoryId: dockerRepository.id,
+      cursor,
+    },
   })
 
-  if (!data || loading) return null
+  useEffect(() => {
+    if (data?.dockerImages) {
```

Comment:
Is this just going to fetch the entire set of images until it stops?  We should only do it when a user scrolls into view for sure, some of these repos have thousands of images

Reply:
I think in designs for this there was a "view more" button.